test/lang.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+/**
+ * Tests using externally scraped websites. May fail if resource at
+ * location changes.
+ */
+
+var meta = require('../index');
+var assert = require('./utils/assert.js');
+
+// mocha defines to avoid JSHint breakage
+/* global describe, it, before, beforeEach, after, afterEach */
+
+describe('scraping', function() {
```

Comment:
You should either change remove or change this description, or move this test to the scraping file. Otherwise when the tests are run, we won't be able to tell where this test lives, because this is the same top level description as in scraping.js test file.